pkg/client/restclient/OWNERS

```diff
@@ -0,0 +1,24 @@
+reviewers:
```

Comment:
@apelisse could you add me to the reviewers of this file? Seems the files are not editable.